taipy/gui/viselements.json

```diff
@@ -51,7 +51,7 @@
                     {
                         "name": "on_action",
                         "type": "Callback",
-                        "doc": "The name of a function that is triggered when the button is pressed.<br/>The parameters of that function are all optional:\n<ul>\n<li>state (<code>State^</code>): the state instance.</li>\n<li>id (optional[str]): the identifier of the button.</li>\n<li>payload (dict): a dictionary that contains the key \"action\" set to the name of the action that triggered this callback.</li>\n</ul>",
+                        "doc": "The name of a function that is triggered when the button is pressed.<br/>The parameters of that function are all optional:\n<ul>\n<li>state (<code>State^</code>): the state instance.</li>\n<li>id (str|None): the identifier of the button.</li>\n<li>payload (dict): a dictionary that contains the key \"action\" set to the name of the action that triggered this callback.</li>\n</ul>",
```

Comment:
I liked the `optional`, I think it's more clear for the user

Reply:
The problem is with the Page Builder API.
type1 | type2 is a valid Python construct that, when injected in the function signature (see https://github.com/Avaiga/taipy/issues/1632), can be interpreted.

Reply:
not for python 3.8 ?

Reply:
Bloody good point. Let's discuss that